modules/coreos-layering-configuring-on.adoc

```diff
@@ -53,30 +51,31 @@ NAME                                       PREPARED   BUILDING   SUCCEEDED   INT
 layered-c8765e26ebc87e1e17a7d6e0a78e8bae   False      False      False        False        True
 ----
 
-// Not in 4.18; maybe 4.19
-// You can manually rebuild your custom layered image by either modifying your `MachineOSConfig` object or applying an annotation to the `MachineOSConfig` object as discussed in the following section.
+You can manually rebuild your custom layered image by either modifying your `MachineOSConfig` object or applying an annotation to the `MachineOSConfig` object as discussed in the following section. For more information, see "Rebuilding an on-cluster custom layered image".
```

Comment:
PEER REVIEW This file was copied directly from [main](https://github.com/openshift/openshift-docs/blob/main/modules/coreos-layering-configuring-on.adoc?plain=1#L70C1-L75C1) (and 4.19). No need for review here.